File: examples/java-maven-publish/src/test/java/com/myorg/CalculatorTest.java
package com.myorg;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

class CalculatorTest {
    @Test
    void add() {
        assertEquals(5, Calculator.add(2, 3));
    }

    @Test
    void greet() {
        assertEquals("Hello, world!", Calculator.greet("world"));
    }
}

File: examples/kotlin-publish/src/test/kotlin/com/myorg/CalculatorTest.kt
package com.myorg

import kotlin.test.Test
import kotlin.test.assertEquals

class CalculatorTest {
    @Test
    fun add() {
        assertEquals(5, add(2, 3))
    }

    @Test
    fun greet() {
        assertEquals("Hello, world!", greet("world"))
    }
}
